systemtest/src/test/java/io/strimzi/systemtest/KafkaST.java

```diff
@@ -512,79 +514,86 @@ void testSendMessagesTlsScramSha() {
         checkPings(messagesCount, job);
     }
 
-    @Test
     @Tag(REGRESSION)
-    void testJvmAndResources() {
+    @ParameterizedTest
+    @CsvSource({
+            "1.1Gi, 1181116006400m, 300m, 300m, 1g, 1g",
+            "1G, 1G, 0.3, 300m, 1G, 1G",
+            "1024M, 1024M, 0.5, 500m, 1024M, 1024M",
+            "1024Mi, 1Gi, 500m, 500m, 1024m, 1024m"
```

Comment:
It might be interesting to use more variability: such as `500Mi` for memory. Also, using different limits / requests for Kafka, Zoo and EO might help you to test more things in one run.

Reply:
yeah, good point with different limits / requests for Kafka, Zoo and EO, but TBH I was frightened by a large number of method arguments... But on the other hand, different values for Kafka and & will save our execution time